services/ui-src/src/provider-mocks/fullStoreMock.js

```diff
@@ -3358,6 +3358,10 @@ const fullStoreMock = {
         state_name: "Alaska",
         state_id: "AK"
       },
+      {
+        label: "American Samoa",
```

Comment:
These are improperly formatted. I'm not sure we want to make the change, but the tests are failing because they should be `{state_name:"", value:""}` in this array